src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/ValidationSummaryTagHelper.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Infrastructure
+{
+    [HtmlTargetElement(TagHelperConstants.Div, Attributes = TagHelperName)]
+    public sealed class ValidationSummaryTagHelper : TagHelper
+    {
+        public const string TagHelperName = "nhs-validation-summary";
+        public const string TitleName = "title";
```

Comment:
These constants could be `private` or `internal` if they might used in tests at a later stage but as with the `TagHelperConstants` there's obviously no harm in them being public.